hackathonbaobab2020/tests/tests.py

```diff
@@ -6,6 +6,7 @@
 import unittest
 import shutil
 from hackathonbaobab2020 import solve_zip, Experiment
+from hackathonbaobab2020 import HackathonApp
```

Comment:
Can't these two imports be done in one statement

Reply:
yes, my bad